engineering_tools/carma

```diff
@@ -75,11 +99,16 @@ carma-config__edit() {
     fi
 
     echo "Opening shell inside carma-config container with read/write privileges..."
+    if [ "$1" = "-d" ]; then
```

Comment:
I'm not sure the flags here (and on inspect) make sense either, these functions are just for using non-carma software to look at and/or edit the CARMA config contents. All that these functions really require is that there is **a** version of carma-base installed so that we can guarantee vim/emacs/sed/awk/etc are available for manipulating the config files. It should never really matter which version of carma is being used for that.